src/pybind/rados/rados.pyx

```diff
@@ -2536,6 +2539,80 @@ cdef class Ioctx(object):
             raise make_ex(ret, "error reading %s" % object_name)
         return completion
 
+    @requires(('object_name', str_type), ('xattr_name', str_type),
+              ('xattr_value', bytes), ('oncomplete', opt(Callable)))
+    def aio_set_xattr(self, object_name, xattr_name, xattr_value, oncomplete = None):
```

Comment:
please consider changing `aio_set_xattr` to `aio_setxattr` looking into librados code, that seems to be more relevant, I'd suggest following the same convention in all your work. Thanks.

Reply:
```
    @requires(('xattr_name', str_type), ('xattr_value', bytes))
    def set_xattr(self, xattr_name, xattr_value):
        """
        Set an xattr
        :param xattr_name : name of the xattr
        :type xattr_name : strgg
        :param xattr_value: buffer to set xattr to
        :type xattr_value: bytes
        """
        xattr_name = cstr(xattr_name, 'xattr_name')
        cdef:
            char *_xattr_name = xattr_name
            char *_xattr_value = xattr_value
            size_t _xattr_value_len = len(xattr_value)
        with nogil:
            rados_write_op_setxattr(self.write_op, _xattr_name, _xattr_value, _xattr_value_len)
```
this is my previous commit,you recommend me to chang setxattr() to set_xattr().
 I'm so confused today.

Reply:
apologies for the same, but taking a closer look into source it'd be more consistent to have the same name of function in python and C.